beacon-chain/rpc/validator/proposer.go

```diff
@@ -248,6 +248,14 @@ func (vs *Server) eth1DataMajorityVote(ctx context.Context, slot uint64) (*ethpb
 		log.WithError(err).Error("Failed to get block number for previous voting period")
 		return vs.randomETH1DataVote(ctx)
 	}
+	eth1FollowDistance := int64(params.BeaconConfig().Eth1FollowDistance)
+	firstValidBlockNumber := big.NewInt(0).Sub(previousPeriodBlockNumber, big.NewInt(eth1FollowDistance))
```

Comment:
`firstValidBlockNumber` is a confusing name, for what is the previous period. Maybe `previousValidBlock` or something like that. The same applies to `lastValidBlockNumber` to `currentValidBlock` or something similar.

Reply:
I changed the names to `currentPeriodInitialBlock` and `previousPeriodInitialBlock`.